PySDM/physics/saturation_vapour_pressure/bolton_1980.py

```diff
@@ -0,0 +1,28 @@
+"""
+[Bolton 1980](https://doi.org/10.1175/1520-0493(1980)108<1046:TCOEPT>2.0.CO;2)
+"""
+
+import numpy as np
+
+
+class Bolton1980:
+    def __init__(self, _):
+        pass
+
+    @staticmethod
+    def pvs_Celsius(const, T):
+        """valid for 0 <= T <= 100 C, eq (9)"""
```

Comment:
this seems to be the Wexler formula. Bolton's fit is eq. (10)?
Worth to rename this to Wexler_1976, and implement eq. (10) here

Reply:
we have to implement only equation 10 or also include with equation 9

Reply:
Eq. (10) is needed to reproduce what was in the J&N paper, and it will help us to nail down where the differences in final drop radii originate from.

Since you've already implemented eq. (9), I suggest keeping it as a separate `Wexler_1976` class in a separate file (that paper has >400 citations in Google Scholar - seems a common choice, worth to have it in PySDM).